flatpak/org.nanuc.Axolotl.yml

```diff
@@ -88,3 +58,38 @@ modules:
           --install.offline true
           --run.offline true
         dest-filename: .yarnrc
+    modules:
+      - name: abseil
```

Comment:
What does abseil do?

Reply:
I think abseil is a dependency of protoc to compile the protobufs

Reply:
Exactly. Previously, we used `git` which also downloaded Git submodules. As we now download via HTTP, we have to download the dependencies separately.